From 93abceb5418be8fc57c595e017b7937454852b8a Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 16 Jan 2024 08:52:05 +0100
Subject: [PATCH 0874/1141] arm64: defconfig: add display connector & ad7511 in
 addons fragment

Enable the display connector driver (link to HDMI) & the hdmi bridge
adv7511.

Change-Id: Ia69f25c5b6cebbb9ba6e5e2983b00dc87235dae7
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/351446
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 279734728925..22f9b98bca77 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -15,7 +15,9 @@ CONFIG_CRYPTO_DEV_STM32_CRYP=m
 CONFIG_CRYPTO_DEV_STM32_HASH=m
 CONFIG_DRM=y
 CONFIG_DRM_BRIDGE=y
+CONFIG_DRM_DISPLAY_CONNECTOR=y
 CONFIG_DRM_DW_MIPI_DSI=y
+CONFIG_DRM_I2C_ADV7511=y
 CONFIG_DRM_KMS_HELPER=y
 CONFIG_DRM_MIPI_DSI=y
 CONFIG_DRM_PANEL=y
-- 
2.39.2

